GOVERNANCE.md

```diff
@@ -0,0 +1,201 @@
+the organization.
+
+The Core Team should have occasional meetings to discuss the project, with meeting minutes
+being added to [eslint-community/governance][].
+
+The Core Team are together with the ESLint TSC the organization owners and the only ones
+with `admin` access to repositories within the organization.
+
+The Core Team are the only members of the `@eslint-community/core-team` team.
+
+The Core Team and ESLint TSC have a shared `#community-team` chat room, in the ESLint
+Discord, to communicate with one another.
+
+## ESLint TSC
+
+The ESLint TSC are the ultimate caretakers of the ESLint-Community, appoints the Core
+Team and holds the ownership of the `eslint-community-bot` npm account that owns all
+the ESLint-Community npm modules.
+
+The Core Team are together with the ESLint TSC the organization owners and only ones
+with `admin` access to repositories within the organization.
+
+The ESLint TSC are the only members of the `@eslint-community/eslint-tsc` team.
+
+The Core Team and ESLint TSC have a shared `#community-team` chat room, in the ESLint
+Discord, to communicate with one another.
+
+## Collaborators
+
+Collaborators maintain the projects of the ESLint-Community organization.
+
+Collaborators are added on a project by project basis by adding them to a team of
+the same name as its main project repository, eg. `@eslint-community/eslint-plugin`.
+
+A project team is given `write` level access to its repositories.
+
+If multiple repositores are needed by the same project they should all be added to
+the project team.
+
+Both Collaborators and non-Collaborators may propose changes to the source code
+of the projects of the organization. The mechanism to propose such a change is a
+GitHub pull request. Collaborators review and merge (_land_) pull requests
+following the [CONTRIBUTING](CONTRIBUTING.md) guidelines.
+
+Collaborators can communicate in private with the Core Team in
+[private team discussion on GitHub][].
+
+### Collaborator activities
+
+* Helping users and novice contributors
+* Contributing code and documentation changes that improve the project
+* Reviewing and commenting on issues and pull requests
+* Merging pull requests
+* Release plugins
+
+The Core Team can remove inactive Collaborators or provide them with
+_Past Collaborators_ status. Past Collaborators may request that the Core
+Team restore them to active status.
+
+## Projects
+
+There are two kinds of projects in ESLint Community: Primary and auxiliary projects.
+
+Primary projects are maintained by Core Team in collaboration with project Collaborators
+and will be ensured to be kept up to date at a best effort pace by the Core Team.
+
+Auxiliary projects are maintained by its project Collaborators and is kept up to date at
+whatever pace those Collaborators chose. The Core Team merely fascilitates nominations
+of new Collaborators to such projects, ensuring there's a path to keep it maintained,
+without making any promises.
```

Comment:
I would maybe want a third level here:

* Archived projects

And a project can move up and down – from primary, to auxiliary to archived and the other way around – and its clear to everyone what the status of the project is and that makes the threshold for accepting new projects lower.

Reply:
I think the wording used here is a bit vague and could use some clarification. For instance, what does "kept up to date" mean?

Here's what I'd suggest:

* **Active Projects** - these are projects under active development, meaning they continue to add new features in addition to bug fixes. These may be maintained either by the core team or separate maintainers. The expectation is that these projects have regular releases.
* **Maintenance Projects** - these are projects not under active development, but they receive critical bug fixes and security updates (i.e., dependency upgrades), as well as any changes to remain compatible with new releases of ESLint. The expectation is that these releases happen infrequently. In most cases, the core team will be handling these projects.
* **EOL Projects** - these are projects that receive no further updates of any kind. They are not maintained, the repo is archived and the npm package is marked as deprecated. These projects are kept purely for historical purposes.

Reply:
I like that suggestion! The wording is much clearer than my suggestion 🙏